src/shared/libfido2-util.c

```diff
@@ -633,6 +638,11 @@ int fido2_generate_hmac_hash(
         if (r == FIDO_ERR_PIN_AUTH_BLOCKED)
                 return log_notice_errno(SYNTHETIC_ERRNO(EPERM),
                                         "Token PIN is currently blocked, please remove and reinsert token.");
+#ifdef FIDO_ERR_UV_BLOCKED
+        if (r == FIDO_ERR_UV_BLOCKED)
+                return log_notice_errno(SYNTHETIC_ERRNO(EPERM),
+                                        "Token verification is currently blocked, please remove and reinsert token.");
```

Comment:
It bugs me a bit that those messages are different here and above…

Reply:
sorry, compared to which ones?

Reply:
Above in this patch. Try this view: https://github.com/systemd/systemd/pull/19298/files/e17429a2998e3600d9f0874bb65d571018fe2335..c5fd3f8da40409ea9bca2c068ce8100f26c8ef2f#diff-bf30439d1f70cf4825bc72a9707c7c157000b8dcc6c2cd4d936aea7519536f4a

Reply:
Yeah, the original ones are different too, it seemed better to keep the messages consistent with the area